File: vitepress/tutorial/scala/ShowNameOfSelectedElement.scala
import com.telelogic.rhapsody.core._

object ShowNameOfSelectedElement extends App {
	val ra: IRPApplication = RhapsodyAppServer.getActiveRhapsodyApplication()
	val se: IRPModelElement = ra.getSelectedElement()
	if (null != se) {
		printf("%s\n", se.getName())
	}
}

File: vitepress/public/tutorial/java/ShowNameOfSelectedElement.java
import com.telelogic.rhapsody.core.*;

public class ShowNameOfSelectedElement {
	public static void main(String[] args) {
		IRPApplication app = RhapsodyAppServer.getActiveRhapsodyApplication();
		IRPModelElement se = app.getSelectedElement();
		if (null != se) {
			System.out.println(se.getName());
		}
	}
}